<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="FluxTest" tests="1" skipped="0" failures="0" errors="0" timestamp="2022-05-11T17:41:07" hostname="DESKTOP-SQQC16J" time="0.24">
  <properties/>
  <testcase name="fluxSubscriberNumbersError()" classname="FluxTest" time="0.24"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[[Test worker] INFO reactor.Flux.Range.1 - | onSubscribe([Synchronous Fuseable] FluxRange.RangeSubscription)
[Test worker] INFO reactor.Flux.Range.1 - | request(3)
[Test worker] INFO reactor.Flux.Range.1 - | onNext(1)
[Test worker] INFO FluxTest - Number 1
[Test worker] INFO reactor.Flux.Range.1 - | onNext(2)
[Test worker] INFO FluxTest - Number 2
[Test worker] INFO reactor.Flux.Range.1 - | onNext(3)
[Test worker] INFO FluxTest - Number 3
[Test worker] INFO FluxTest - -----------------------------------
[Test worker] INFO reactor.Flux.Range.1 - | onSubscribe([Synchronous Fuseable] FluxRange.RangeSubscription)
[Test worker] INFO reactor.Flux.Range.1 - | request(unbounded)
[Test worker] INFO reactor.Flux.Range.1 - | onNext(1)
[Test worker] INFO reactor.Flux.Range.1 - | onNext(2)
[Test worker] INFO reactor.Flux.Range.1 - | onNext(3)
[Test worker] INFO reactor.Flux.Range.1 - | onNext(4)
[Test worker] INFO reactor.Flux.Range.1 - | cancel()
]]></system-err>
</testsuite>
